upf/upf_pfcp.c

```diff
@@ -2417,22 +2417,27 @@ pfcp_lookup_cp_f_seid (pfcp_ie_f_seid_t *f_seid)
 }
 
 static int
-urr_increment_and_check_counter (u64 *packets, u64 *bytes, u64 *consumed,
-                                 u64 threshold, u64 quota, u64 n_bytes)
+urr_increment_and_check_counter (urr_volume_t *volume, u8 urr_status,
```

Comment:
typedef enum this u8 urr_status?

Reply:
agree, but let's postpone it to keep scope for bugfix smaller